integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -649,19 +649,21 @@ csi:
         requests:
           cpu: 10m
           memory: 20Mi
-  # Will run fuse daemon inside csi nodeserver
+  # Run alluxio fuse process inside csi nodeserver container if mountInPod = false
+  # Run alluxio fuse process inside a separate pod if mountInPod = true
+  mountInPod: false
   nodePlugin:
     hostNetwork: true
     dnsPolicy: ClusterFirstWithHostNet
     nodeserver:
       resources:
-        # The default xmx is 8G
+        # fuse in nodeserver container needs more resources
         limits:
-          cpu: "4"
-          memory: "8G"
+          cpu: 4
+          memory: 8G
         requests:
-          cpu: "1"
-          memory: "1G"
+          cpu: 10m
+          memory: 300Mi
```

Comment:
I guess this is a FUSE daemon which is a JVM? Will a low request and high limit cause the JVM heap resize? Would it be better to allocate more resources at first? Did you get this 300Mi from some test or estimation?

Reply:
If it's modal based on `mountInPod` I'd stick with the default being the greedy request, and then conditionally use some lower value in the template file (i.e: `if .Values.csi.mountInPod`).

Reply:
Makes a lot of sense. Thanks

Reply:
Let's just keep the resources as they were. If there's no fuse process in the container, 1 cpu and 1G memory is probably excessive but I don't think it's gonna hurt